Applications/_Plugins/CMesh/mesh_optimizer/mesh_optimizer.cpp

```diff
@@ -39,10 +39,11 @@ SET_PLUGIN_NAME("TOOTLE_MESH")
 void *make_Plugin_Mesh_Optimizer() { return new Plugin_Mesh_Optimizer; }
 #endif
 
+namespace cmesh_mesh_opt {
```

Comment:
This were namespaces because when statically linking these plugins we get a linker error for duplicate symbols. I feel there is a big lack of namespacing in the project. All plugins and major libraries should likely live in their own unique namespaces.